apps/sensenet/cypress/integration/dashboard/dashboard.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+const repoUrl = Cypress.env('repoUrl')
+
+describe('Dashboard', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl } }))
+  })
+  it('Header should contains the url of the current repository.', () => {
+    cy.get('[data-test="sensenet-header"]').contains(repoUrl)
+  })
+
+  it(`should have title`, () => {
+    cy.get('h1').contains(/Welcome to your [a-z0-9]+ project/)
+  })
+
+  it('Subscription section should have the Developer plan text and features list', () => {
+    expect('3 users').to.match(/[0-3] users/)
+    expect('500 content').to.match(/[0-500] content/)
```

Comment:
please add a `data-test` attribute to the element (as you did with the header) and select the element by this attribute